src/daemon/control.rs

```diff
@@ -846,12 +846,13 @@ pub fn check_spend_transaction_size(revaultd: &RevaultD, spend_tx: SpendTransact
     }
 
     let deposit_outpoints: Vec<OutPoint> = tx.input.iter().map(|i| i.previous_output).collect();
-    let tx_hex = encode::serialize_hex(&tx);
+    let tx = base64::encode(encode::serialize(&tx));
```

Comment:
`.to_string()`?

Reply:
tx.to_string() doesn't work, and putting tx directly in the serde macro doesn't work either (not serialized in base64, it's just printed in a human friendly way like "inputs: [], outputs: []" etc)